.prettierignore

```diff
@@ -1,4 +1,17 @@
 code-examples/protect-page-login/php/composer.lock
 CONTRIBUTING.md
-.github/pull_request_template.md
+.github/*
+tests/*
+src/*
+docs/cli/*
+docs/hydra/cli/*
+docs/keto/cli/*
+docs/kratos/cli/*
+docs/oathkeeper/cli/*
 package-lock.json
+.prettierignore
+.prettierrc.js
+docs/markdownlint.yml
+docusaurus.config.js
+jest.config.js
+playwright.config.ts
```

Comment:
These config files are edited by humans. Let's keep them prettified so that there are no more discussions on how they are formatted!